README.md

```diff
@@ -1 +1,7 @@
-# Design_Pattern
\ No newline at end of file
+# Design_Pattern
+
+List of Design Patterns:
+
+1.Factory Pattern
+2.Decorator Design Pattern
+3.Observer Design Pattern
```

Comment:
How is the observer used in the phone?

Reply:
Phone class will be implementing an interface Observer(update(),addObserver(),removeObserver()).
Observer interface will also be implemented into the Crickbuzz(getScore()) class which will update the score.
Crickbuzz class will containing a <list of observers> for which the notification will be send.